compose/install.md

```diff
@@ -100,13 +131,22 @@ sudo pip install docker-compose
 ### Install as a container
 
 Compose can also be run inside a container, from a small bash script wrapper.
-To install compose as a container run:
+To install compose as a container run this command. Be sure to replace the version number with the one that you want, if this example is out-of-date:
 
 ```bash
-$ curl -L --fail https://github.com/docker/compose/releases/download/1.12.0/run.sh > /usr/local/bin/docker-compose
+$ curl -L --fail https://github.com/docker/compose/releases/download/1.13.0/run.sh > /usr/local/bin/docker-compose
```

Comment:
This is another place you could use the front-matter variablee as I suggested above.